resources/qml/dialogs/RoomSettings.qml

```diff
@@ -152,171 +165,191 @@ ApplicationWindow {
+                Button {
+                    text: qsTr("Configure")
+                    ToolTip.text: qsTr("Select events to hide in this room")
+                    onClicked: hiddenEventsDialog.show()
+                    Layout.alignment: Qt.AlignRight
+                }
 
-            Button {
-                text: qsTr("Configure")
-                ToolTip.text: qsTr("Select events to hide in this room")
-                onClicked: hiddenEventsDialog.show()
-                Layout.alignment: Qt.AlignRight
-            }
+                Item {
+                    // for adding extra space between sections
+                    Layout.fillWidth: true
+                }
 
-            Item {
-                // for adding extra space between sections
-                Layout.fillWidth: true
-            }
+                Item {
+                    // for adding extra space between sections
+                    Layout.fillWidth: true
+                }
 
-            Item {
-                // for adding extra space between sections
-                Layout.fillWidth: true
-            }
+                Label {
+                    text: qsTr("INFO")
+                    font.bold: true
+                }
 
-            MatrixText {
-                text: qsTr("INFO")
-                font.bold: true
-            }
+                Item {
+                    Layout.fillWidth: true
+                }
 
-            Item {
-                Layout.fillWidth: true
-            }
+                Label {
+                    text: qsTr("Internal ID")
+                }
 
-            MatrixText {
-                text: qsTr("Internal ID")
-            }
+                Label {
+                    text: roomSettings.roomId
+                    font.pixelSize: Math.floor(fontMetrics.font.pixelSize * 0.8)
+                    wrapMode: Text.WrapAnywhere
+                    Layout.alignment: Qt.AlignRight
+                    Layout.fillWidth: true
+                }
 
-            MatrixText {
-                text: roomSettings.roomId
-                font.pixelSize: Math.floor(fontMetrics.font.pixelSize * 0.8)
-                Layout.alignment: Qt.AlignRight
-            }
+                Label {
+                    text: qsTr("Room Version")
+                }
 
-            MatrixText {
-                text: qsTr("Room Version")
-            }
+                Label {
+                    text: roomSettings.roomVersion
+                    font.pixelSize: fontMetrics.font.pixelSize
+                    Layout.alignment: Qt.AlignRight
+                }
 
-            MatrixText {
-                text: roomSettings.roomVersion
-                font.pixelSize: fontMetrics.font.pixelSize
-                Layout.alignment: Qt.AlignRight
             }
-
         }
-
-        DialogButtonBox {
-            Layout.fillWidth: true
-            standardButtons: DialogButtonBox.Ok
-            onAccepted: close()
+    }
+    Button {
```

Comment:
Imo this should just be a child or so of the topic, for me this just sticks to the bottom of the room settings.

Reply:
![grafik](https://user-images.githubusercontent.com/16312324/156479951-44e50797-ebfe-424e-8885-3c861753060a.png)

Reply:
I just have no button now

![image](https://user-images.githubusercontent.com/16312324/156669160-4ba0b264-0144-4afb-a88a-295f0c08ef23.png)